index.html

```diff
@@ -201,7 +206,7 @@ <h2 class="work_title">Multi-Post Stories</h2>
             </div>
         </section>
         <section class="about">
-                <div>
+                <div class="desktop-version">
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see your design is `inconsistent` and looks unprofessional on the desktop version >=768px, your about section content is not aligned properly. Please have a look at the below-attached image and table.


### Design Comparison
<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 19-59-21](https://github.com/Afsheen96/Portfolio/assets/66476329/2aa756b9-a7eb-4087-8eb6-c63356bda891)

</td>
<td>

![Screenshot from 2023-06-10 19-59-41](https://github.com/Afsheen96/Portfolio/assets/66476329/69cf5b4d-db3d-4a40-8341-99a682376975)

</td>
</tr>
</table>

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

Reply:
Kindly fix the `about section` of your portfolio to be similar to the figma design given 

Figma design :
![image](https://github.com/Afsheen96/Portfolio/assets/20537712/82a39a03-78fa-48b6-802f-48bd682675ac)

design given :
![image](https://github.com/Afsheen96/Portfolio/assets/20537712/fa4f972c-4241-4c65-889b-03ddfc46a34c)